File: teams/Alina_Petrosyan/c++/15_12_2021/fibonacci.cpp
#include <iostream>

int fibonacci(int num) {
        if(num == 1) {
                return 0;
        }
        if(num == 2) {
                return 1;
        }
        return fibonacci(num - 1) + fibonacci(num -2);
}       

int main() {
	int num;
        std::cout << "Please input the number of fibonacci sequence: ";
	std::cin >> num;
	std::cout << "The " << num << "th fibonacci number is: " << fibonacci(num) << std::endl;
	return 0;
}

File: teams/Vardan_Parzyan/Python/31_01_22/Fibonacci.py
# # RECURSION
def fibonacci(num):
    if num <=2:
        return 1
    else:
        return fibonacci(num-1) + fibonacci(num -2)
a = int(input("Enter your number : "))
print(fibonacci(a))

# LOOP
def fibonacciLoop(num):
    if num <=2:
        return 1
    num1 = 0
    num2 = 1
    fib = 0
    for x in range(num-1):
        fib = num1 + num2
        num1 = num2
        num2 = fib
    return fib
a = int(input("Enter your number : "))
print(fibonacciLoop(a))